tests/test_aaa_authorization_service.rb

```diff
@@ -92,6 +95,14 @@ def tacacs_groups
     %w(tac_group bxb100 sjc200 rtp10)
   end
 
+  def valid_cfg?(cfg)
```

Comment:
I would like for us to take this one step further and actually verify reach-ability to the tacacs server and skip if tests if it's not reachable.  Ruby has a builtin ping that looks simple enough.
```
irb(main):001:0> require 'ping'
=> true
irb(main):002:0> Ping.pingecho('dt-n9k5-1', 10)  <---- Valid Host
=> true
irb(main):003:0> Ping.pingecho('dt-n9k5-2', 10) <---- Invalid Host
=> false
irb(main):004:0> 
```